doc/reference/autoinstall-reference.rst

```diff
@@ -846,12 +845,12 @@ package
 
 The name of the package, e.g., ``linux-image-5.13.0-40-generic``
 
-flavor
+:spellexception:`flavor`
 ~~~~~~
 
 * **type:** string
 
-The flavor of the kernel, e.g., ``generic`` or ``hwe``.
+The flavour of the kernel, e.g., ``generic`` or ``hwe``.
```

Comment:
There is a problem in this document where we are trying to use en-GB spellings of words when the schema requires en-US ones.  What is the best way to handle this problem?  Toggling back and forth between the spellings doesn't sound prudent.

Reply:
>  What is the best way to handle this problem?

Honestly? Canonical should stop demanding that docs conform to British spelling :) IT English is overwhelmingly US English.

At this point, I see no way around it, as silly as it looks.

Reply:
OK, we'll have to endure.